packages/tradershub/src/features/cfd/modals/TradeModal/TradeModal.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { FC, useEffect } from 'react';
+import QRCode from 'qrcode.react';
+import { Provider } from '@deriv/library';
+import { Text, useBreakpoint } from '@deriv/quill-design';
+import { Modal } from '../../../../components/Modal';
+import { THooks, TMarketTypes, TPlatforms } from '../../../../types';
+import { AppToIconMapper, LinksMapper, PlatformDetails, TAppLinks } from '../../constants';
+import { MT5TradeScreen } from '../../screens/MT5TradeScreen';
+
+type TTradeModalProps = {
+    account?: THooks.CtraderAccountsList | THooks.DxtradeAccountsList | THooks.MT5AccountsList;
+    marketType?: TMarketTypes.All;
+    platform: TPlatforms.All;
+};
+
+const TradeModal: FC<TTradeModalProps> = ({ account, marketType, platform }) => {
```

Comment:
I think we can get rid of `FC` let's not normalise it. It doesn't add any value to the component and it just add unnecessary imports from `React`.

```suggestion
const TradeModal = ({ account, marketType, platform }: TTradeModalProps) => {
```